app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,27 @@
+class RentsController < ApplicationController
+
+  #before_action :authenticate_user!
+
+
+  def create
+    rent = Rent.new(create_params.merge(user: current_user) )# with this we can be sure the rent will be associated to the current_user
+    rent.save
+
+    if rent.save
+      mail = RentMailer.rent_email(rent)
+      mail.deliver_later
+      render json: rent
+    else
+      puts "Sad path"
+    end
+  end
+
+  def create_params
+    params_permitted = params.permit(:book_id, :rent_date, :rent_end)
+  end
+
+  def show
+    render_paginated Rent.where('user_id=' + current_user.id.to_s)
```

Comment:
This endpoint should be named `index`, not `show`. According to the convention, `index` is used to render multiple resources, and `show` to render only ONE resource. In this case we want an index.
And remember my correction about concatenating strings, try to avoid it when possible. In this case you could do `render_paginated current_user.rents` (another way is `Rent.where(user: current_user)`)

Reply:
`Rent.where(user_id: 99)`